extension/src/openvic-extension/singletons/MenuSingleton.cpp

```diff
@@ -489,6 +490,32 @@ Dictionary MenuSingleton::get_province_info_from_index(int32_t index) const {
 		ret[province_info_rgo_employment_percentage_key] = (rgo.get_total_employees_count_cache() * fixed_point_t::_100() / max_employee_count).to_float_rounded();
 	}
 
+	String amount_of_employees_by_pop_type;
+	auto const& employee_count_per_type_cache = rgo.get_employee_count_per_type_cache();
+	for (PopType const& pop_type : *employee_count_per_type_cache.get_keys()) {
+		const pop_size_t employees_of_type = employee_count_per_type_cache[pop_type];
+		if (employees_of_type > 0) {
+			amount_of_employees_by_pop_type += Utilities::std_to_godot_string(pop_type.get_identifier().data()) + ": " +
+			Utilities::std_to_godot_string(std::to_string(employees_of_type)) + "\n";
+		}
+	}
+
+	String base_work_force;
+	if (rgo.get_production_type_nullable() == nullptr) {
```

Comment:
rgo has a method called `is_valid()`
Use that instead of null checking.

If the rgo is not valid, don't bother doing any tooltip stuff